lib/polint.rb

```diff
@@ -129,36 +168,18 @@ def check_pair(key, val, contexts, is_fuzzy)
       @errors  += 1 if not_in_val.any? || not_in_key.any? || is_empty
       @fuzzies += 1 if is_fuzzy
 
-      if key =~ /(\d+):(.*)/
-        lineno = $1
-        file_and_line = "#{@pofile}:#{$1}:"
```

Comment:
The only thing that's a little bit misleading is that because `annotate_and_load_po` transforms the file a bit before parsing if the parse step fails the line numbers and columns may be out by up to a few lines which makes parse errors a pain to track down.

I only realised this later on and I think it's possible to remove the need to annotate the file by extracting the line information from Parslet as it parses the file and putting it into the parse tree, but the way I've written the parser and transformer isn't particularly amenable to this because I wrote them before I realised I wanted to do it.

I think the parser/transformer could be written a bit more cleanly (the subtree transformations I've done are a bit messy really) but it was working so wanted to get something in initially and then can always come back to that plus add line info.